### PR TITLE
Change Quick Settings icon

### PR DESCRIPTION
**Pull request by florianduros**
## Checklist

-   [x] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

In preparation of the new Thread Activity Centre (https://github.com/element-hq/wat-internal/issues/93). 
The Quick Settings icon is too small in comparison to the Thread Activity Centre icon and the compound icons.

- Replace the current icon by the `settings-solid` icon [(Figma design)](https://www.figma.com/file/WGM39wEVQSJxnSFf6eeVry/Fixing-Threads-notifs?node-id=732%3A57854&mode=dev)
- Increase the icon size
- The old setting icon is still used in other button of EW

<table>
<tr>
 <td>Old</td>
 <td>
image
<td>
image
</td>
<tr>
 <td>New</td>
 <td>
image
</td>
<td>
image
</td>
</table>

---
Here's what your changelog entry will look like:

## ✨ Features
 * Change Quick Settings icon ([\#12141](https://github.com/matrix-org/matrix-react-sdk/pull/12141)). Contributed by @florianduros.

### CODE REVIEW

File: res/css/structures/_QuickSettingsButton.pcss [hunk 1]
```diff
@@ -36,10 +36,10 @@ limitations under the License.
         width: 32px;
         height: 32px;
         left: 0;
-        mask-image: url("$(res)/img/element-icons/settings.svg");
```
Comment (dbkr) [inline res/css/structures/_QuickSettingsButton.pcss:39]: I assume this is still used somewhere else?

**Review by dbkr — APPROVED** [review]
Looks good, does this need design review or is it fine because, well, it looks like the figma?

File: res/css/structures/_QuickSettingsButton.pcss (hunk 1 above)
Comment (florianduros) [reply res/css/structures/_QuickSettingsButton.pcss:39]: Yes!

**Comment by florianduros** [comment]
@dbkr Yes, I'll wait for @gaelledel review on it.

**Review by gaelledel — APPROVED** [review]
thx

**Pull request merged**